src/index.css

```
@@ -543,3 +565,224 @@ body {
+ * Timeline (6s): compare sweep → swap → green sweep → reset.
+ */
+
+/* Tile 0: compare at 10%, sort at 15% */
+@keyframes proSort0 {
+  0%,
+  10% {
+    background-color: var(--color-primary);
+    border-color: color-mix(in srgb, var(--color-primary) 30%, transparent);
+    transform: translateY(0);
+  }
+  10%,
```

Stylelint (17.14.0)

[error] Duplicate keyframe selector "10%" (keyframe-block-no-duplicate-selectors)

(keyframe-block-no-duplicate-selectors)